disnake/interactions/base.py

```diff
@@ -1531,7 +1549,7 @@ async def edit(
         if attachments is MISSING and (file or files):
             attachments = self.attachments
 
```

Comment:
same thing in this edit method further up, can't comment directly because github

Reply:
I left that wording in this method alone since that snippet is repeated in multiple other locations and I figured the consistent wording made sense.